ingredient/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+from django.http       import JsonResponse
+from django.views      import View
+from recipe.models     import Recipe
+from ingredient.models import Ingredient
 
-# Create your views here.
+
+class ShowIngredientInfo(View):
```

Comment:
class 명은 명사로 만들어주세요! Info를 붙이는 것도 너무 verbose한 것 같습니다!

```suggestion
class Ingredient(View):
```

Reply:
똑같이 Ingredient로 하면 ingredient.models에서 import 한 Ingredient와 혼동이 일어나지는 않을까요>?? 알아서 참조가 잘될까요??

Reply:
`IngredientView`로 수정해주세요~ django convention이에요! View class에 상속받은 class를 붙여준답니다!